apb/tasks/main.yml

```diff
@@ -1,19 +1,11 @@
 ---
-
-# These can always be overridden at the command line
-# http://docs.ansible.com/ansible/devel/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable
 - name: 'Set facts'
   set_fact:
-    cluster: "{{ _cluster_type }}"
-    broker_auto_escalate: "{{ 'true' if _cluster_type == 'kubernetes' else 'false' }}"
-    broker_local_openshift_enabled: "{{ 'true' if _cluster_type == 'openshift' else 'false' }}"
-  vars:
-    _cluster_type: '{{ "openshift" if "openshift" in lookup("k8s", cluster_info="version") else "kubernetes" }}'
+    broker_auto_escalate: "{{ 'true' if 'project.openshift.io' not in lookup('k8s', cluster_info='api_groups') else 'false' }}"
```

Comment:
Based on https://github.com/kubernetes-incubator/service-catalog/commit/97f79ce9088b55cfe153328d171b7239f5464a58

Even in k8s the helm chart will enable the originating identity header now:
https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/catalog/values.yaml#L165

So should we just default this to 'false' for both cases now?

Reply:
> So should we just default this to 'false' for both cases now?

This is what I was thinking, also loudly warning if that header is missing and auto_escalate isn't specified.